src/SmartHotel.Clients/SmartHotel.Clients/Views/HomeView.xaml

```diff
@@ -337,15 +337,19 @@
                         Grid.Column="1"
                         BackgroundColor="{StaticResource ChartBackgroundColor}">
                         <Grid.RowDefinitions>
-                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="*" />
                             <RowDefinition Height="*" />
                         </Grid.RowDefinitions>
-						<microcharts:ChartView Chart="{Binding LightChart}"
-											   Margin="16, 36, 16, 6" />
+
                         <Label     
-                            Grid.Row="0"
                             Text="{extensions:Translate AmbientLight}"
                             Style="{StaticResource ChartTextStyle}"/>
+                        
```

Comment:
how come you switched the position of the chart and the label in code?

Reply:
With "Auto" it looks like that 
![image](https://user-images.githubusercontent.com/8102322/46157445-03c86c80-c27c-11e8-9fe0-1bc74ae0b56c.png)

with "*" :

![image](https://user-images.githubusercontent.com/8102322/46157518-2fe3ed80-c27c-11e8-8fdd-e1fa66d95510.png)